build.gradle.kts

```
@@ -271,6 +279,10 @@ allprojects {
                     attributes["Specification-Title"] = "JSVG"
                     attributes["Implementation-Vendor"] = "JSVG"
                     attributes["Implementation-Vendor-Id"] = "com.github.weisj"
+                    // bnd instructions to generate OSGi metadata
+                    attributes["Bundle-SymbolicName"] = "com.github.weisj.jsvg"
+                    attributes["-exportcontents"] = "!*.impl.*,*"
+                    attributes["-removeheaders"] = "Private-Package,Tool"
                 }
```

Adding the bnd instruction to a separate bundle task (?) here
```suggestion
                }
                bundle {
                    bnd("""
Bundle-SymbolicName: com.github.weisj.jsvg
-exportcontents: !*.impl.*,*
-removeheaders: Private-Package,Tool
""")
                }
```
lead to a build failure with:
```
Could not determine the dependencies of task ':annotations:assemble'.
> Could not create task ':annotations:sourcesJar'.
   > Extension with name 'bundle' does not exist. Currently registered extension names: [ext]
```